src/messages/components/__tests__/MessageManagerContainer.spec.tsx

```diff
@@ -58,7 +58,7 @@ describe('Message Manager component', () => {
         format: MessageFormat.IN_PAGE,
         locations: [Location.UniProtKBResults],
       }),
-      route: '/uniprotkb?query=blah',
+      route: '/uniprotkb?=query=blah',
```

Comment:
This `?=` feels wrong, could you double check why this is needed, or if it's just a typo that just slipped in without any effect on the test